### PR TITLE
Create model and first data

### PR DESCRIPTION
**Pull request by stg101**
- Model for tasks has been created
- Some data has been added

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -1 +1,22 @@
-console.log("I'm alive");
+tasksArray = [
```
Comment (luciagirasoles) [inline assets/js/main.js]: If this is our "seed".. I might suggest to add that info here

File: assets/js/main.js [hunk 2]
```diff
@@ -1 +1,22 @@
...
+  },
+  {
+    description: "Update Cuchi",
+    creation_date: new Date("2019-01-01"),
+    due_date: new Date("2019-01-02"),
+    marked: false
+  }
+];
+
+console.log(tasksArray);
```
Comment (cristianbgp) [inline assets/js/main.js]: Remember to delete the `console.log();`

File: assets/js/main.js (hunk 1 above)
Comment (stg101) [reply assets/js/main.js]: Good idea, I already added some comments.

File: assets/js/main.js (hunk 2 above)
Comment (stg101) [reply assets/js/main.js]: I need this for debugging

**Pull request merged**